input/postgres/relations.go

```diff
@@ -242,6 +248,14 @@ func GetRelations(ctx context.Context, db *sql.DB, postgresVersion state.Postgre
 
 		row.DatabaseOid = currentDatabaseOid
 
+		bufferCache, ok := ts.BufferCache[currentDatabaseOid]
+		if ok {
+			row.CachedDataBytes = bufferCache[dataFilenode]
+			row.CachedToastBytes = bufferCache[toastFilenode]
+			bufferCache[dataFilenode] = 0
+			bufferCache[toastFilenode] = 0
```

Comment:
Maybe that comment should be here instead? And a code comment rather than just a PR comment? I like the approach, but I, too, was confused on first reading this.